OXRS-SHA-StateIO-ESP32-FW.ino

```diff
@@ -182,13 +209,21 @@ void setConfigSchema()
   
   JsonObject ioConfig = json.createNestedObject("ioConfig");
   ioConfig["type"] = "string";
+  ioConfig["description"] = "Hint ! Restart your device when changed to make effective!";
```

Comment:
Can I suggest changing these to "A restart is required before changes will take effect".

Reply:
Thank's for the sugestion. It was my expectation that you would come up with a better wording.